scripts/deploy-prod.sh

```
@@ -0,0 +1,21 @@
+set -euo pipefail
+
+VPS_HOST="72.62.117.123"
+VPS_PATH="/opt/ping-mem"
+
+echo "Syncing to VPS..."
+rsync -av --exclude='.env' --exclude='node_modules' --exclude='.git' \
+  /Users/umasankr/Projects/ping-mem/ \
+  root@${VPS_HOST}:${VPS_PATH}/
+
+echo "Patching port for VPS (3003 -> 3000)..."
+ssh root@${VPS_HOST} "sed -i 's/127.0.0.1:3003:3003/127.0.0.1:3000:3000/g; s/PING_MEM_PORT=3003/PING_MEM_PORT=3000/g' ${VPS_PATH}/docker-compose.prod.yml"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)